pages/community.tsx

```diff
@@ -0,0 +1,397 @@
+
+const TRACKING_CATEGORY = "community";
+
+const SEO = {
+  title: "Join the Best Web3 Community | Learn, Build, Grow",
+  description:
+    "Join the thirdweb community and share your web3 journey alongside 35,000+ other builders. Learn, build, & grow with thirdweb.",
+};
+
+const CommunitySections = [
+  {
+    title1: "Learn",
+    title2: "web3 development",
+    description:
+      "Whether you're interested in web3 or thirdweb, our mission is to cultivate a community of like-minded individuals who share our passion for the future of decentralized technology.",
+
+    cards: [
+      {
+        image: "",
+        link: "/learn",
+        title: "thirdweb learn",
+        description: "",
+      },
+      // {
+      //   image: "",
+      //   link: "/learn",
+      //   title: "Bootcamp",
+      // description: ""
+      // },
+      {
+        image: "",
+        link: "/events",
+        title: "Events",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/events",
+        title: "Office Hours",
+        description: "",
+      },
+    ],
+  },
+  {
+    title1: "Build",
+    title2: "innovative projects",
+    description:
+      "From hackathons to bounties, open source to community projects, there are so many ways to build no matter your level of contribution. ",
+    cards: [
+      {
+        image: "",
+        link: "/events",
+        title: "Hackathons",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://www.notion.so/thirdweb/Paid-Projects-e090e7cefdc14c0f8e8ad6aa47dbdb74?pvs=4",
+        title: "Projects",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://thirdweb.notion.site/thirdweb-Bug-Bounty-Program-f78d1db776ab4f0e974c9da176fcf706",
+        title: "Bug Bounty",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/oss",
+        title: "OSS Contributions",
+        description: "",
+      },
+    ],
+  },
+  {
+    title1: "Join",
+    title2: "the community",
+    description:
+      "Explore programs to educate, host events, and showcase your achievements - together, we can take the web3 community to new heights!",
+    cards: [
+      {
+        image: "",
+        link: "https://ambassador.thirdweb.com/",
+        title: "Ambassadors",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://www.notion.so/thirdweb/University-Program-Page-e090e7cefdc14c0f8e8ad6aa47dbdb74?pvs=4",
+        title: "University Program",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://www.notion.so/thirdweb/Regional-Champions-Page-cebe9db7b2534162b0fd87e60039280c?pvs=4",
+        title: "Regional Champions",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/",
+        title: "Showcase",
+        description: "",
+      },
+    ],
+  },
+];
+
+const galleryImages = [
```

Comment:
can we make this any other way that makes more sense? the image names are sequential this seems like a lot of repetition, also they should use require and then use ChakraNextImage for image optimization